tests/prepare/install/data/debuginfo.fmf

```diff
@@ -1,39 +1,23 @@
-summary:
-    Install debuginfo packages
+summary: Install debuginfo packages
+
 prepare:
-  - name: debuginfo
     how: install
-    package:
-      - grep-debuginfo
-      - elfutils-debuginfod-debuginfo
```

Comment:
+1 to use just grep-debuginfo however the `elfutils-debuginfod-debuginfo` seems to be test for correct striping of the 'debuginfo' suffix. Could we have that coverd by unit test?

Reply:
Ok, added in 7231d4f6.